src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_common.py

```diff
@@ -71,17 +72,15 @@ def firewall_rule_create_func(client, resource_group_name, server_name, firewall
 
 
 def firewall_rule_delete_func(client, resource_group_name, server_name, firewall_rule_name, yes=None):
-    confirm = yes
     result = None
     if not yes:
-        confirm = user_confirmation(
+        user_confirmation(
```

Comment:
why add this change?
I think the following delete operation will be run only when we get positive confirm, so need to check confirmation value as previous design.
Add @DaeunYim for confirmation.

Reply:
`user_confirmation` will raise exceptions if not positive confirm, so no need to check the return value. Once the function returns successfully, it must be positive confirm.

Reply:
If there is no confirmation, `user_confirmation` raise an exception,If confirmed, the following program will execute normally, so there is no effect.

Reply:
Checked and confirmed. Thanks @Juliehzl for the heads up.